### PR TITLE
Use consistent-env instead of consistent-path

### PR DESCRIPTION
**Pull request by steelbrain**
We will not just be figuring out the `PATH` but all the tokens of keys or configurations the user might have set for applications in their rc files

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -82,24 +82,24 @@ export function validateFind(directory: string, name: string | Array<string>) {
...
 ): Promise<ExecResult> {
   const options: ExecOptions = assign({
-    env: assign({}, process.env),
+    env: {},
     stream: 'stdout',
     throwOnStdErr: true
   }, opts)
 
-  if (isNode && options.env) {
+  options.env = assign(await consistentEnv.async(), options.env)
```
Comment (Arcanemagus) [inline src/helpers.js]: Why move this down here? Does it need to be for the `await`?

Comment (steelbrain) [inline src/helpers.js]: Good question

Comment (steelbrain) [inline src/helpers.js]: Done

**Comment by Arcanemagus** [comment]
LGTM, pending build success :wink:.

**Pull request merged**